larray/viewer/api.py

```diff
@@ -65,25 +65,33 @@ def get_title(obj, depth=0, maxnames=3):
 
 def edit(obj=None, title='', minvalue=None, maxvalue=None, readonly=False, depth=0):
     """
-    Opens a new editor window. If no object is given,
-    all local arrays are loaded in the editor.
+    Opens a new editor window.
 
     obj : np.ndarray, LArray, Session, dict or str, optional
-        Object to visualize. If string, array(s) will be loaded
-        from the file given as argument.
-        Defaults to the collection of all local variables where
-        the function was called.
+        Object to visualize. If string, array(s) will be loaded from the file given as argument.
+        Defaults to the collection of all local variables where the function was called.
     title : str, optional
-        Title for the current object.
-        A default one is generated if not provided.
+        Title for the current object. Defaults to the name of the first object found in the caller namespace which
+        corresponds to `obj` (it will use a combination of the 3 first names if several names correspond to the same
+        object).
```

Comment:
Remember that, if a MappingEditor is created, a new title will be generated by the method `update_title` of MappingEditor which looks like 'filepath - array's name : axes_info - passed title'.